sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiWriteUnshardedRecords.java

```diff
@@ -1164,5 +1164,10 @@ public void teardown() {
         throw new RuntimeException(e);
       }
     }
+
+    @Override
+    public Duration getAllowedTimestampSkew() {
+      return Duration.millis(Long.MAX_VALUE);
+    }
```

Comment:
StorageApiWritesShardedRecords has 

```
    @Override
    public Duration getAllowedTimestampSkew() {
      return Duration.millis(BoundedWindow.TIMESTAMP_MAX_VALUE.getMillis());
    }
```

I think the semantics are the same, just to confirm

Reply:
Yup that value is equal to Long.MAX_VALUE

Reply:
can we keep the consistency?

Reply:
Done